File: 1-two-sum/two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        result = [0,0]
        dict_ = {}
        for i in range(len(nums)):
            if(target-nums[i] in dict_):
                return [i, dict_[target - nums[i]]]
            dict_[nums[i]] = i

File: 1-two-sum/two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        HashMap<Integer,Integer> hp = new HashMap();
        int [] result = new int[2];
        for(int i =0;i<nums.length;i++){
            if(hp.containsKey(target - nums[i])){
                result[0] = hp.get(target - nums[i]);
                result[1] = i;
        }
            hp.put(nums[i], i);
        }
        return result;
    }
}